src/presentational-components/platform/platform-card.tsx

```diff
@@ -16,12 +17,17 @@ import CatalogLink from '../../smart-components/common/catalog-link';
 import { StyledCard } from '../styled-components/styled-gallery';
 import { StyledCardBody } from '../styled-components/card';
 import CardIcon from '../shared/card-icon';
+import { DateFormat } from '@redhat-cloud-services/frontend-components/components/cjs/DateFormat';
+import labelMessages from '../../messages/labels.messages';
+import useFormatMessage from '../../utilities/use-format-message';
 
 const TO_DISPLAY = ['description', 'modified'];
 
 export interface PlatformCardProps extends ItemDetailsProps {
   name: string;
   id: string;
+  availability_status: string;
+  last_successful_refresh_at: string;
```

Comment:
if the prop is not required the definition should be `availability_status?: string`